src/css/_TextSelection.scss

```diff
@@ -0,0 +1,41 @@
+.textSelectionSVG {
+    width: 100%;
+    position: absolute;
+    height: 100%;
+    top: 0;
+    left: 0;
+    pointer-events: none;
+
+    .BRwordElement {
+        pointer-events: all;
+    }
+
+    // Highlight colors for text selection layer ( these 2 properties do not work if joined)
+    .BRwordElement::selection {
+        background: hsla(210, 74%, 62%, 0.4);
+    }
+    .BRwordElement::-moz-selection {
+        background: hsla(210, 74%, 62%, 0.4);
+        color: transparent;
+    }
+
+    .BRparagElement {
+        fill: red;
```

Comment:
Is this needed?

```suggestion
```

Reply:
Red color useful when testing, should I remove it?

Reply:
Since it's not being used, yeah; it's a little confusing.